scripts/provision.sh

```diff
@@ -408,19 +348,16 @@ create_users() {
 }
 
 if [ "$#" -eq 0 ]; then
-    check_envfile
     start_init
     start_builder
   else
     for arg in "$@"
     do
       if [ "$arg" == "--help" ] || [ "$arg" == "-h" ]; then
-	    Help
-      elif [ "$arg" == "--generate-bootstrap" ]; then
-	    generate_frontend_bootstrap_bundle
+	      Help
       elif [ "$arg" == "--install-frontend" ]; then
         export FRONTEND_INSTALL=1
-	    start_frontend_from_bootstrap_bundle
+	      install_frontend
```

Comment:
install_frontend call should align with the line above.

Reply:
ahh. looks like a tabs vs. spaces issue